pkg/cmd/pulumi/config.go

```diff
@@ -913,17 +913,24 @@ func looksLikeSecret(k config.Key, v string) bool {
 // getStackConfiguration loads configuration information for a given stack. If stackConfigFile is non empty,
 // it is uses instead of the default configuration file for the stack
 func getStackConfiguration(stack backend.Stack, sm secrets.Manager) (backend.StackConfiguration, error) {
+	var cfg config.Map
 	workspaceStack, err := loadProjectStack(stack)
 	if err != nil {
-		return backend.StackConfiguration{}, fmt.Errorf("loading stack configuration: %w", err)
+		// if project is unavailable, fallback to get the project's configuration from the backend
+		cfg, err = backend.GetLatestConfiguration(commandContext(), stack)
+		if err != nil {
+			return backend.StackConfiguration{}, fmt.Errorf("loading stack configuration: %w", err)
```

Comment:
Nit: perhaps combine the two errors somehow (from loadProjectStack and backend.GetLatestConfiguration). Currently only the second error is reported. Or if there's logging around could use verbose logging for the ignored error.